pkg/event/factory.go

```diff
@@ -37,7 +37,22 @@ const botFilteringKey = "$opt_bot_filtering"
 const revenueKey = "revenue"
 const valueKey = "value"
 
+func getEventEndPoint(region, eventEndPoint string) string {
+	if eventEndPoint != "" {
+		return eventEndPoint
+	}
+	switch region {
+	case "EU":
+		return EventEndPoints["EU"]
+	case "US":
+		return EventEndPoints["US"]
+	default:
+		return EventEndPoints["US"]
+	}
+}
+
```

Comment:
Also are wexpecting usrrs may enter lowercase "us"? if so then you could add:
region = strings.ToUpper(region)
unles we restrict to uppercase only